deb-package-builder/extensions/mongo/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ex
+source ${DEB_BUILDER_DIR}/functions.sh
+
+echo "Building mongo for gcp-php${SHORT_VERSION}"
+
+PNAME="gcp-php${SHORT_VERSION}-mongo"
+
+# Download the source
+if [ ${SHORT_VERSION} == '70' ] || [ ${SHORT_VERSION} == '71' ]; then
```

Comment:
Maybe reverse this to only build for PHP 5.6? We also build extensions for 7.2 as well, so this will likely crash for 7.2 builds.